<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>982</width>
    <height>538</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>宋体</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>学生管理系统</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>541</width>
      <height>214</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true">alternate-background-color: rgb(211, 249, 248);</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>75</number>
       </attribute>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpenDB"/>
   <addaction name="separator"/>
   <addaction name="action_2"/>
   <addaction name="action_3"/>
   <addaction name="action_4"/>
   <addaction name="actRecInsert"/>
   <addaction name="actRecEdit"/>
   <addaction name="actRecDelete"/>
   <addaction name="separator"/>
   <addaction name="action"/>
   <addaction name="actQuit"/>
  </widget>
  <action name="actOpenDB">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/open3.bmp</normaloff>:/images/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>查看课程信息</string>
   </property>
   <property name="toolTip">
    <string>打开数据</string>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
  </action>
  <action name="actQuit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/exit.bmp</normaloff>:/images/images/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
  </action>
  <action name="actRecInsert">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/306.bmp</normaloff>:/images/images/306.bmp</iconset>
   </property>
   <property name="text">
    <string>添加课程</string>
   </property>
   <property name="toolTip">
    <string>插入记录</string>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
  </action>
  <action name="actRecDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/delete1.bmp</normaloff>:/images/images/delete1.bmp</iconset>
   </property>
   <property name="text">
    <string>删除课程</string>
   </property>
   <property name="toolTip">
    <string>删除记录</string>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
  </action>
  <action name="actRecEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/812.bmp</normaloff>:/images/images/812.bmp</iconset>
   </property>
   <property name="text">
    <string>编辑课程</string>
   </property>
   <property name="toolTip">
    <string>编辑记录</string>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
  </action>
  <action name="actSQLtest">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/322.bmp</normaloff>:/images/images/322.bmp</iconset>
   </property>
   <property name="text">
    <string>SQL测试</string>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
  </action>
  <action name="action">
   <property name="icon">
    <iconset>
     <selectedon>:/images/images/00.JPG</selectedon>
    </iconset>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
   <property name="toolTip">
    <string>输入编号查找对应学号</string>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/open3.bmp</normaloff>:/images/images/open3.bmp</iconset>
   </property>
   <property name="text">
    <string>查看学生信息</string>
   </property>
   <property name="toolTip">
    <string>学生信息</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/704.bmp</normaloff>:/images/images/704.bmp</iconset>
   </property>
   <property name="text">
    <string>学生选课</string>
   </property>
   <property name="toolTip">
    <string>选课</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/706.bmp</normaloff>:/images/images/706.bmp</iconset>
   </property>
   <property name="text">
    <string>学生退课</string>
   </property>
   <property name="toolTip">
    <string>退课</string>
   </property>
   <property name="menuRole">
    <enum>QAction::TextHeuristicRole</enum>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
